service/src/main/java/com/ge/predix/acs/encryption/AcsEncryption.java

```diff
@@ -0,0 +1,69 @@
+package com.ge.predix.acs.encryption;
+
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import org.apache.commons.codec.binary.Base64;
+
+import com.google.common.primitives.Bytes;
+
+public class AcsEncryption {
+
+    private String key;
+
+    private static final String ALGO = "AES/CBC/PKCS5PADDING";
+
+    public AcsEncryption(final String key) {
+        this.key = key;
+    }
+
+    public String encrypt(final String value) {
+        try {
+            if (this.key.length() != 24) {
+                throw new RuntimeException("Encryption key must be string of length 24");
+            }
+            byte[] ivBytes = new byte[16];
+            SecureRandom.getInstanceStrong().nextBytes(ivBytes);
+            IvParameterSpec iv = new IvParameterSpec(ivBytes);
+            SecretKeySpec skeySpec = new SecretKeySpec(key.getBytes("UTF-8"), "AES");
+
+            Cipher cipher = Cipher.getInstance(ALGO);
+            cipher.init(Cipher.ENCRYPT_MODE, skeySpec, iv);
+
+            byte[] encrypted = cipher.doFinal(value.getBytes());
+            byte[] result = Bytes.concat(ivBytes, encrypted);
+
+            return Base64.encodeBase64String(result);
+        } catch (Exception ex) {
```

Comment:
* catch `Throwable` to be safe
* pls remove the printStackTrace
* log a error (excluding the encryption key)